### PR TITLE
Added validation for cross-company contract allocation

### PR DESCRIPTION
**Pull request by leifbjarte**

### CODE REVIEW

File: src/backend/api/Fusion.Resources.Database/Entities/DbContract.cs [hunk 1]
```diff
@@ -15,6 +16,14 @@ public class DbContract
         public DateTimeOffset Allocated { get; set; }
         public DbPerson AllocatedBy { get; set; } = null!;
         public Guid AllocatedById { get; set; }
-    }
 
+        public string CompanyName { get; set; } = null!;
```
Comment (HansDahle) [inline src/backend/api/Fusion.Resources.Database/Entities/DbContract.cs:20]: Company name would not be sufficient.. I suggest doing a lookup to the contract using the resolver, on the fly.

Comment (leifbjarte) [reply src/backend/api/Fusion.Resources.Database/Entities/DbContract.cs:20]: Ok, will change it.

File: src/backend/api/Fusion.Resources.Domain/Commands/Contracts/AllocateContract.cs [hunk 2]
```diff
@@ -96,7 +96,7 @@ private async Task<DbContract> EnsureDbContract(AllocateContract command, DbProj
                         OrgContractId = contract.Id,
                         ProjectId = dbProject.Id,
                         Allocated = DateTimeOffset.UtcNow,
-                        AllocatedBy = command.Editor.Person
+                        AllocatedBy = command.Editor.Person,
```
Comment (HansDahle) [inline src/backend/api/Fusion.Resources.Domain/Commands/Contracts/AllocateContract.cs:99]: Trailing comma

**Comment by HansDahle** [comment]
@leifbjarte  Any reason not to merge the branch? Have Mia tested this, since we are live now? 
Should ask her to verify that manage personnel still works as expected in QA before morving in to prod.

**Comment by leifbjarte** [comment]
> @leifbjarte Any reason not to merge the branch? Have Mia tested this, since we are live now?
> Should ask her to verify that manage personnel still works as expected in QA before morving in to prod.

No reason, just didn't want to merge it right before easter. Merging now.

**Pull request merged**